File: spring-best/src/main/java/com/best/springbest/model/Shop.java
package com.best.springbest.model;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.DateTimeFormat;

import javax.persistence.*;
import java.util.Date;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "shops")
public class Shop {
    @Id
    private int id;

    private String name;

    private String image;

    private String address;

    private String email;

    private String phone;

}

File: BEST-App-main/lib/model/shop.dart
class Shop {
  final int id;
  final String name;
  final String image;
  final String address;
  final String email;
  final String phone;

  Shop(this.id, this.name,this.image ,this.address, this.email, this.phone);

  static Map<String, dynamic> toJson(Shop shop) {
    return {
      'id': shop.id,
      'name': shop.name,
      'image': shop.image,
      'address': shop.address,
      'email': shop.email,
      'phone': shop.phone
    };
  }
}
